src/modules/auth/auth.controller.ts

```diff
@@ -0,0 +1,94 @@
+import { Controller, Post, Body, Param, Get, Res, HttpStatus } from '@nestjs/common';
+import {
+	ApiBadRequestResponse,
+	ApiNotFoundResponse,
+	ApiOkResponse,
+	ApiOperation,
+	ApiParam,
+	ApiResponse,
+	ApiTags,
+	ApiUnauthorizedResponse,
+} from '@nestjs/swagger';
+import express from 'express';
+import { z } from 'zod';
+
+import { TranslateService } from '@modules/translate/translate.service';
+import { User } from '@modules/users/entities/user.entity';
+import { UsersDataService } from '@modules/users/services/users-data.service';
+import { validate } from '@utils/validate';
+
+import { AuthService } from './auth.service';
+import { UserPostDTO } from './dto/register.dto';
+import { UserSignInDTO } from './dto/sign-in.dto';
+import { TokenDTO } from './dto/token.dto';
+
+@ApiTags('Authentification')
+@Controller('auth')
+export class AuthController {
+	constructor(
+		private readonly t: TranslateService,
+		private readonly authService: AuthService,
+		private readonly userService: UsersDataService,
+	) {}
+
+	@Post('login')
+	@ApiOperation({ summary: 'Sign in a user with email and password' })
+	@ApiUnauthorizedResponse({ description: 'Unauthorized, password invalid' })
+	@ApiNotFoundResponse({ description: 'User not found' })
+	@ApiOkResponse({ description: 'OK', type: TokenDTO })
```

Comment:
It is better practice to refer only to http code. In this case the status 200 is clear enough to not send body with the request.